From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@remirepo.net>
Date: Mon, 13 Sep 2021 14:59:47 +0200
Subject: [PATCH 2/3] add comment to not remove these macros

---
 src/gd.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gd.h b/src/gd.h
index 1ad9e63..71f5a89 100644
--- a/src/gd.h
+++ b/src/gd.h
@@ -1604,6 +1604,8 @@ BGD_DECLARE(void) gdImageFlipHorizontal(gdImagePtr im);
 BGD_DECLARE(void) gdImageFlipVertical(gdImagePtr im);
 BGD_DECLARE(void) gdImageFlipBoth(gdImagePtr im);
 
+/* Macros still used in gd extension up to PHP 8.0
+   so please keep these unused macros for now */
 #define GD_FLIP_HORINZONTAL 1 /* typo, kept for BC */
 #define GD_FLIP_HORIZONTAL 1
 #define GD_FLIP_VERTICAL 2
